### ISSUE TITLE
nip46 error should not be `null`

### ISSUE THREAD

**Issue by alexgleason** [greenart7c3/Amber#38]
It is possible for Amber to send a response like:

```json
{"error":null,"id":"23af55b0-698c-4150-a56a-9a508c658ce3","result":"ack"}
```

This causes my serializer to crash:

```
error: Uncaught (in promise) ZodError: [
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "null",
    "path": [
      "error"
    ],
    "message": "Expected string, received null"
  }
]
```

The result should look like this (error is omitted):

```json
{"id":"23af55b0-698c-4150-a56a-9a508c658ce3","result":"ack"}
```

**Comment by alexgleason** [comment]
Fixed by 518a610c92d6bc61bdc8a0ea106b9215aba1f0f2

@greenart7c3 Can you do a release please? Thank you for fixing these! :clap:

**Comment by greenart7c3** [comment]
Sure, i'll make a release later
